rc-events-service/src/main/scala/com/epam/random_coffee/events/repo/EventRepository.scala

```diff
@@ -0,0 +1,19 @@
+package com.epam.random_coffee.events.repo
+
+import com.epam.random_coffee.events.api.request.UpdateEventRequest
+import com.epam.random_coffee.events.api.response.EventResponse
+import com.epam.random_coffee.events.model.Event
+
+import scala.concurrent.Future
+
+trait EventRepository {
+  def save(eventName: String): Future[Unit]
+
+  def findByName(eventName: String): Future[Option[EventResponse]]
```

Comment:
Seems we should return here (at repository level) Future[Option[Event]] and then map it to EventResponse in higher level (services level). For convenient  mapping use companion object for EventResponse.

Reply:
I deleted findByName method, because it isn't needed now.